src/test/java/dk/cngroup/lentils/repository/FinalPlaceRepositoryTest.java

```diff
@@ -48,25 +48,24 @@ public void findByIdTest() {
     }
 
     @Test(expected = javax.validation.ConstraintViolationException.class)
-    public void finalPlaceWithEmptyTitleTest() {
-        FinalPlace finalPlace = new FinalPlace(TEST_EMPTY_TITLE, TEST_LOCATION, null);
+    public void finalPlaceWithEmptyDescriptionTest() {
+        FinalPlace finalPlace = new FinalPlace(TEST_EMPTY_DESCRIPTION, TEST_LOCATION, null);
         repository.saveAndFlush(finalPlace);
     }
 
     @Test(expected = javax.validation.ConstraintViolationException.class)
-    public void finalPlaceWith256TitleTest() {
-        FinalPlace finalPlace = new FinalPlace(TEST_256_TITLE, TEST_LOCATION, null);
+    public void finalPlaceWithDescriptionLengthOver1000Test() {
```

Comment:
Obecne neni dobry napad davat do nazvu hodnoty tech konstant -> 1000.

Co kdyz se jednou nekdo rozhodne zvetsit to na dvojnasobek. Cislo neni platne a mate. A porad to opravovat te zacne otravovat :)

Takze jsem spis pro finalPlaceWithTooLongDescriptionTest